multinet/multinet/entity.py

```diff
@@ -0,0 +1,43 @@
+from . import db
+from .types import *
```

Comment:
I'm not a fan of `import *` because it makes it harder to cross-reference names in the file later. I'd prefer if this were an explicit import of the names used, or just `from . import types`.

Reply:
Now that we have smaller scopes, this cleaned up nicely.